aopy/data/bmi3d.py

```diff
@@ -1117,6 +1117,91 @@ def get_ts_data_segment(preproc_dir, subject, te_id, date, start_time, end_time,
 
     return data, samplerate
 
+def get_spike_data_segment(preproc_dir, subject, te_id, date, port, start_time, end_time, bin_width=.01):
```

Comment:
for future compatability with tomo's PR, can you change port to drive and make it an optional argument? you could just have drive=1 for now by default